docs/DeveloperGuide.md

```diff
@@ -293,6 +293,36 @@ The following sequence diagram illustrates this flow:
 
 ![NusmodsScrapeModuleSequenceDiagram](images/NusmodsScrapeModuleSequenceDiagram.png)
 
+### Command Stalling feature
+
+GradPad stalls certain commands that mutate/erase data so that users can provide confirmation as to
```

Comment:
nit: since it's only clear and delete that are stalled (not edit), should we remove mutate?

Reply:
oh ya hor ok will change!